Jenkinsfile

```diff
@@ -80,7 +75,10 @@ node('linux && docker') {
       }
 
       stage('Deployment pipeline upload') {
-        sh 'deployment-package package create --with-deploy || true'
+        lerna = readJSON file: 'lerna.json'
+        prereleaseVersion = lerna.version
+        version = prereleaseVersion.split('-alpha')[0]
```

Comment:
How do you see supporting hotfix deployments, so as to avoid overriding the "latest" entry in the cdn or publishing the wrong version to master/npm? (more a discussion for a future PR)